src/AddressRegistry.Api.BackOffice.Handlers.Sqs/Handlers/SqsAddressCorrectRejectionHandler.cs

```diff
@@ -0,0 +1,44 @@
+namespace AddressRegistry.Api.BackOffice.Handlers.Sqs.Handlers
+{
+    using Abstractions;
+    using Requests;
+    using System.Collections.Generic;
+    using TicketingService.Abstractions;
+
+    public sealed class SqsAddressCorrectRejectionHandler : SqsHandler<SqsAddressCorrectRejectionRequest>
+    {
+        public const string Action = "CorrectAddressFromRejectedToProposed";
```

Comment:
Please change to something in the likes of https://github.com/Informatievlaanderen/address-registry/blob/9ba880c2ee4b087d2eecc568f096e9b97f1c90b0/src/AddressRegistry.Api.BackOffice.Handlers.Sqs/Handlers/SqsAddressCorrectRetirementHandler.cs#L10